cmake_modules/functions.cmake

```diff
@@ -1,7 +1,7 @@
 
 function( findRpiRevision OUTPUT )
 	# Find it with an automated script
-	execute_process( COMMAND grep -Po "^Revision\\s*:\\s*\\K[[:xdigit:]]+" /proc/cpuinfo OUTPUT_VARIABLE TMP )
+	execute_process( COMMAND grep -Po "^Revision\\s*:\\s*\\K[[:xdigit:]]+" /proc/cpuinfo OUTPUT_VARIABLE TMP OUTPUT_STRIP_TRAILING_WHITESPACE )
```

Comment:
Without this flag, the variable contains a trailing newline, which is inherently yucky.